app/views/users/show.html.erb

```diff
@@ -1,6 +1,10 @@
 <h1><%= @user.full_name %></h1>
 
-<% if @user.id == current_user.id %>
+<% if admin? %>
+  <%= link_to "manage users", admin_users_path %>
+<% end %>
+
+<% if @user.id == session[:user_id] %>
```

Comment:
I changed this line (:3 in the old version, :7 in the new) because we can save a hit to the database